src/components/contentLibrary/contentLibrary.component.tsx

```diff
@@ -40,54 +56,99 @@ export const ContentLibrary = () => {
     [objectListingWidth, windowSize],
   );
 
+  const sensors = useSensors(
+    useSensor(PointerSensor, {
+      activationConstraint: {
+        delay: 250,
+        tolerance: 5,
+      },
+    }),
+  );
+
   return (
-    <div
-      className="flex h-screen flex-row"
-      style={{
-        maxHeight: `calc(100vh - 5rem)`,
-      }}
+    <DndContext
+      onDragStart={handleDragStart}
+      onDragEnd={handleDragEnd}
+      sensors={sensors}
     >
-      <m.div
-        ref={objectListingRef}
-        className={clsx(
-          "w-full max-w-full pt-6 pl-2 md:pl-6 lg:pl-10",
-          activePanelObject && "md:w-1/2 lg:w-5/12 xl:w-3/5",
-        )}
+      <div
+        className="flex h-screen flex-row"
         style={{
-          width:
-            activePanelObject && objectListingWidth
-              ? objectListingWidth
-              : "100%",
+          maxHeight: `calc(100vh - 5rem)`,
         }}
       >
-        <ObjectList
-          withCreateButtons
-          onInfoClick={setActivePanelObject}
-          isPanelOpen={!!activePanelObject}
-        />
-      </m.div>
-
-      {activePanelObject && (
-        <m.div className="fixed z-50 flex h-full w-full grow flex-row bg-white drop-shadow-md md:relative md:z-auto lg:drop-shadow-none">
-          <m.div
-            data-testid="drag-bar"
-            key={windowSize}
-            className="hidden w-3 cursor-col-resize items-center bg-manatee-100 lg:flex "
-            onDrag={handleDrag}
-            drag="x"
-            dragConstraints={{ top: 0, left: 0, right: 0, bottom: 0 }}
-            dragElastic={0}
-            dragMomentum={false}
-          >
-            <div className="mx-1 h-20 w-full rounded bg-manatee-600"></div>
-          </m.div>
-          <Panel
-            closePanel={() => setActivePanelObject(null)}
-            uid={activePanelObject.uid}
-            objectType={activePanelObject.objectType}
+        <m.div
```

Comment:
I don't understand why this is in a `m.div` now

Reply:
this was already there. is for the drag from framer to resize when panel opens